src/plugins/intel_gna/tests/unit/transformations/gna_insert_copy_layer.cpp

```diff
@@ -153,8 +161,12 @@ class InsertCopyLayerSplitConcatTest : public InsertCopyLayerTest {
             auto split = ngraph::builder::makeSplit(params, ngraph::element::i64, m_inputs_num, m_axis);
 
             ngraph::OutputVector concat_inputs;
+            int copy_layer_interval =
```

Comment:
how does 
``` 
(Limitations::get_instance()->get_memory_alignment() / Limitations::kBytesPerSplitElement) *
                m_inputs_num / input_shape[0];
````
 match 

```
(m_inputs_num / 8)
```

Reply:
copy_layer_interval defines to which input the copy layer should be added
example:
test vector: inputCounts( here m_inputs_num) = {64, 128, 256};

m_inputs_num = 64: 
(m_inputs_num / 8) = 8,
(64/2)*m_inputs_num/265 = 8
...
m_inputs_num = 256: 
(m_inputs_num / 8) = 32,
(64/2)*m_inputs_num/265 = 32